jina/peapods/pods/k8slib/kubernetes_tools.py

```diff
@@ -83,7 +85,10 @@ def networking_v1_beta1_api(self):
 __k8s_clients = K8SClients()
 
 
-def create(template: str, params: Dict, custom_resource_dir: Optional[str] = None):
+def create(template: str, 
+    params: Dict,
+    logger: JinaLogger,
```

Comment:
Maybe make this optional and if not provided use `default_logger` ? That construct is a bit ugly, but might be good to not force the caller here always to create their own logger

Reply:
SGTM, will update it.